packages/helpers/test/__fixtures__/asyncapi-websocket-query.yml

```
@@ -67,4 +74,21 @@ components:
+    channel:
+      $ref: '#/channels/marketDataV1'
+    messages:
+      - $ref: '#/channels/marketDataV1/messages/subscribeMarket'
+
+  receiveMarketUpdate:
+    summary: Receive real-time market data updates.
+    action: receive
+    channel:
+      $ref: '#/channels/marketDataV1'
+    messages:
+      - $ref: '#/channels/marketDataV1/messages/marketUpdate'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)